### DISCUSSION TITLE
yolov5n vs yolov5s

### DISCUSSION THREAD

**Discussion by antimo22** [ultralytics/yolov5#6259] [Q&A]
Hi everybody, I'm using yolov5 models and I was trying to **find the differences between yolov5n and yolov5s** in terms of architecture, because I would like to cite them in my thesis work. Can anyone help me? The only thing that I'm able to notice is the difference in _depth_multiple_ and _width_multiple_, but I dont' understand how they affect the model. Thanks in advice.

**Comment by glenn-jocher** [comment]
@antimo22 these are two model scaling parameters that I adopted from EfficientDet. They are used to scale the model in depth and width. See https://arxiv.org/abs/1911.09070

**Reply by antimo22** [reply]
Clear. Thank you for your time!

**Comment by PraveenMNaik** [comment]
Sir, is there any particular case that we need to use one of yolov5(s,n,m,l)... does it depend on the number of images taken?...

**Comment by glenn-jocher** [comment]
@PraveenMNaik any model can be used on any dataset. Smaller models are faster but less accurate.

**Comment by ahmadsakir** [comment]
Sir, how do i convert .pt to .ptl   or   .tflite?

**Reply by Heatdh** [reply]
You can pass to onnx then export to tflite. 
Pytorch (pt) => ONNX => TFlite
